rusoto/services/cur/src/generated.rs

```diff
@@ -14,59 +14,51 @@ use rusoto_core::signature::SignedRequest;
 use serde_json::Value as SerdeJsonValue;
 use serde_json::from_str;
 #[doc="Region of customer S3 bucket."]
-pub type AWSRegion = String;
 #[doc="Enable support for Redshift and/or QuickSight."]
```

Comment:
We aren't generating the type aliases any more, but the comments for them remain in the generated code.  Those should get cut too.